tests/test_strategy.py

```diff
@@ -242,18 +244,20 @@ def test_select_action_logic_corner_cases(a_list_p, a_list_cost):
     }
 
     c = CostControlBandit(subsidy_factor=1)
-    # if cost factor is 1 => return the action with min cost
-    assert min(actions_cost, key=actions_cost.get) == c.select_action(p=p, actions=actions)
+    # if cost factor is 1 => return the action with min cost and max probability in case
```

Comment:
Suggestion for this comment (as it is in the code):

`# select the action with the cheapest cost (and the highest probability in case of cost equality)`